### ISSUE TITLE
Flaky e2e test `Add new district`

### ISSUE THREAD

**Issue by m5r** [medic/cht-core#7968]
**Describe the issue**
`Add new district tests :  should edit district with contact_type` sometimes fails with the following error message:
```
Error: Can't call setValue on element with selector "[name="/data/district_hospital/name"]" because element wasn't found
     at implicitWait (/home/runner/work/cht-core/cht-core/node_modules/webdriverio/build/utils/implicitWait.js:34:19)
     at async Element.elementErrorHandlerCallbackFn (/home/runner/work/cht-core/cht-core/node_modules/webdriverio/build/middlewares.js:20:29)
     at async Element.wrapCommandFn (/home/runner/work/cht-core/cht-core/node_modules/@wdio/utils/build/shim.js:131:29)
     at async Object.editDistrict (/home/runner/work/cht-core/cht-core/tests/page-objects/default/contacts/contacts.wdio.page.js:232:3)
     at async Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/contacts/add-new-district.wdio-spec.js:75:5)
```

[Example run](https://github.com/medic/cht-core/actions/runs/3628093421/jobs/6121291128#step:15:566)

**Describe the improvement you'd like**
Stabilize the test

**Describe alternatives you've considered**
None

**Comment by garethbowen** [comment]
Success rate as of today: 92%

**Comment by tatilepizs** [comment]
Fixed on PR #8720
